### ISSUE TITLE
2.1.0 beta 1 on linux: HTML producer starts after about 1 minute

### ISSUE THREAD

**Issue by krzyc** [CasparCG/Server#523]
On latest build 2.1.0.294 a8775b6 Beta 1 running on Ubuntu 16.10 I have a problem with HTML producer.

After issuing CG 1 ADD 1 "template" I see results after about 1 minute. Then everything works as expected, even after CG 1 CLEAR. If I run Caspar, wait few minutes, and then issue CG 1 ADD... I still have to wait about 1 minute before I see HTML template running.

On Windows build HTML templates work as expected from start.

[caspar_log.txt](https://github.com/CasparCG/Server/files/650389/caspar_log.txt)

There are some messages appearing just after issuing command, not included in log:
```
[1213/233311:ERROR:browser_main_loop.cc(214)] GTK theme error: Unable to locate theme engine in module_path: "adwaita",
[1213/233311:ERROR:browser_main_loop.cc(214)] GTK theme error: Unable to locate theme engine in module_path: "adwaita",
[1213/233311:ERROR:renderer_main.cc(226)] Running without renderer sandbox
Fontconfig error: Cannot load default config file
```

There are no messages when template shows up.

**Comment by HellGore** [comment]
I can't reproduce this issue. Do you think it is related to the GTK theme error?

```sudo apt install gnome-themes-standard```

...should install the theme engine.

**Comment by krzyc** [comment]
After installing gnome-themes-standard GTK theme error is gone, but 1 minute freeze is still present.
After downloading latest version from builds - same.
It looks like html file is opened, but freeze happens immediately - than after some time everything returns to normal and works stable.
This problem is not only affecting me, as I have received e-mail today regarding this problem from another user.

What can I do to help you find out where the problem is? What is your linux distribution where this issue is not present - maybe this is Ubuntu related problem only.

```
[2017-03-22 18:39:11.264] [21025] [info]    Sent message to 127.0.0.1:202 CG OK\r\n
[0322/183911:ERROR:renderer_main.cc(226)] Running without renderer sandbox
[0322/183911:INFO:CONSOLE(19)] "init()", source: file:///[...]/score.html (19)
[0322/183911:INFO:CONSOLE(29)] "update()", source: file:///[...]/score.html (29)
[0322/183911:INFO:CONSOLE(22)] "play()", source: file:///[...]/score.html (22)
Fontconfig error: Cannot load default config file
```

**Comment by string-theorist** [comment]
Hi guys, 

I am experiencing the same issue on Ubuntu Studio 16.04. I also get these errors, when it fails to play the html immediately:

`ERROR:renderer_main.cc(226)] Running without renderer sandbox
Fontconfig error: Cannot load default config file`

Eventually it starts working again, but I have noticed it crash again afterwards.

Let me know if there's anything I can do to help as well.

**Comment by string-theorist** [comment]
Notes from what I've tried:

Restarting the server immediately (even multiple times) does not correct the issue.

Sending the LOAD command before the PLAY command doesn't help.

Using a standard web font instead of a local custom font does not correct the issue either. (I've tried using Helvetica). I will try a number of other fonts as well just in case.

**Comment by moschopsuk** [comment]
I've also experience this error on Ubuntu 16.10 desktop.
Its odd that I don't encounter this error on the 16.10 server version if I set up a headless virtual display.

I should so point out that on the desktop its built from source and on the server i'm using the pre-existing binaries.

**Comment by captnced** [comment]
I'm having the same issue with a casparcg server built and running on a Azure (nv6) and Amazon EC2 (g2.xlarge); the OS is ubuntu 16.04 LTS;

**Comment by krzyc** [comment]
@HellGore Can you give me hint on which linux distribution you run it successfuly? Maybe there is a problem with Ubuntu provided libraries, because all reports are Ubuntu related. If you have any idea how this bug can be hunted, please provide it also. This is serious bug, as it affects multiple users. Thanks.

**Comment by HellGore** [comment]
14.04 LTS -- The supported Linux distribution. We have actually never announced support for any other version than that.

**Comment by krzyc** [comment]
It looks that defining FONTCONFIG_PATH solves the problem for me.
`export FONTCONFIG_PATH=/etc/fonts`

**Comment by ronag** [comment]
Should be added to README.

**Comment by ronag** [comment]
@Julusian

**Comment by ronag** [comment]
This should be fixed in 2.2.0. Please re-open if still an issue.
